Sources/FingerprintJSTests/TestDoubles/Spies/IdentifierStorableSpy.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+@testable import FingerprintJS
+
+final class IdentifierStorableSpy: IdentifierStorable {
+    var storeIdentifierInputs: [String: UUID] = [:]
+
+    var loadIdentifierReturnValue: UUID? = nil
+    var loadIdentifierReturnDictionary: [String: UUID] = [:]
+
+    var storeIdentifierCallCount: Int = .zero
+    var loadIdentifierCallCount: Int = .zero
```

Comment:
I would suggest to just stick to `loadIdentifierReturnDictionary`, so as to avoid ambiguities about how the spy is supposed to behave.
```suggestion
    var loadIdentifierReturnDictionary: [String: UUID] = [:]
    
    private(set) var storeIdentifierInputs: [String: UUID] = [:]

    private(set) var storeIdentifierCallCount: Int = .zero
    private(set) var loadIdentifierCallCount: Int = .zero
```